packages/core/src/DdSdkReactNative.tsx

```diff
@@ -70,14 +70,14 @@ export class DdSdkReactNative {
      * @param configuration the configuration for the SDK library
      * @returns a Promise.
      */
-    static async initialize(
+    static initialize = async (
```

Comment:
What's the advantage of putting `async` after?

Reply:
It's a consequence of changing this function from a `function` declaration to an arrow function:

```js
class MyClass {
  // with function declaration
  async myAsyncFunction() { }
  
  // with arrow function
  myAsyncArrowFunction = async () => { }
}
```

The benefit of using arrow functions is that they bind `this` to the class instance, when it can be a bit trickier with function declaration.

You can read more about `this` value in js (not an easy concept) here for instance: https://medium.com/codex/understanding-this-in-javascript-the-complete-guide-c4c21fe15ff8

My rule of thumb is to use arrow functions by default, makes things much easier ^^